src/index.js

```diff
@@ -427,3 +474,15 @@ async function reduceModule(modul, extra = null) {
 
   await Reducer.reduce(modul, additional);
 }
+
+function isValidFile(file) {
```

Comment:
suggest to add a comment to describe this function. For example, something like the following:
Check whether the files are valid. For now, the files with empty extension and starting with "." are not valid. Example invalid files are: .eslintc, .editorconfig, .gitattributes, .npmignore, .hidden.